src/event.ts

```diff
@@ -0,0 +1,11 @@
+import {Text, textMake} from '@toreda/strong-types';
+
+export class Event {
```

Comment:
if i understand the purpose of this class correctly a better name might be StateValue. we use the term event in bifrost meaning a specific thing which differs from here